### PR TITLE
fix(perf-tuning): update GC parameters improvement for Bonita 2021.2

### PR DESCRIPTION
**Pull request by marielle-spiteri**
* This is the recommendations provided by support team to every customer for an improved GC management

### CODE REVIEW

File: modules/runtime/pages/performance-tuning.adoc [hunk 1]
```diff
@@ -399,6 +399,13 @@ Notably, we recommend you to set the initial (`-Xms`) and maximum (`-Xmx`) heap
 This reduces the likelihood of the JVM garbage collector starting. +
 While the garbage collector is running, it prevents creation of new objects, which slows down the application server.
 
+===== Garbage Collector
+We recommend to update the Garbage Collector parameters to use the following ones:  +
```
Comment (alachambre) [inline modules/runtime/pages/performance-tuning.adoc]: Maybe explain why? What a user is going to gain by doing this?

**Comment by marielle-spiteri** [comment]
will only keep #1851

**Comment by tbouffard** [comment]
@mariellebonita I have reopened the Pull Request and merge conflicts to remove the configuration for Java 8.
Please tell me if this is ok, then I will merge the PR.

File: modules/runtime/pages/performance-tuning.adoc (hunk 1 above)
Comment (tbouffard) [reply modules/runtime/pages/performance-tuning.adoc]: this will be improved in #1855

**Comment by tbouffard** [comment]
Putting the PR in draft, we need to merge #1855 first and manage conflicts here once the content of older versions has been merged into `2021.2`.

**Comment by tbouffard** [comment]
Rationale for the change I introduced in 8bc7ecf

before | after
-------- | -----------
image | image

File: modules/runtime/pages/performance-tuning.adoc [hunk 2]
```diff
@@ -400,12 +400,12 @@ This reduces the likelihood of the JVM garbage collector starting. +
...
 
 ===== Garbage Collector
-We recommend to update the Garbage collector parameters to use the following ones (`UseParallelGC` has been proven more eficient than `UseParNewGC` in our experience), depending on wether you use Java 8 or Java 11:  +
+We recommend to update the Garbage Collector parameters to use the following ones (`UseParallelGC` has been proven more eficient than `UseParNewGC` in our experience):
 
 * Edit file `setup/tomcat-templates/setenv.sh`
-* To add in the line `CATALINA_OPTS=`:
-** If you use *Java 8* add: `-XX:+UseParallelGC -XX:ParallelGCThreads=10 -XX:+PrintGCDetails -XX:+PrintGCTimeStamps -Xloggc:${CATALINA_BASE}/logs/gc-$(date +%Y_%m_%d-%H_%M).log`
-** If you use *Java 11* add: `-XX:+UseParallelGC -XX:ParallelGCThreads=10 -Xlog:gc*:file=${CATALINA_BASE}/logs/gc-$(date +%Y_%m_%d-%H_%M).log:time,uptime,hostname,pid:filecount=5,filesize=20M`
+* In the line: `CATALINA_OPTS=`
```
Comment (tbouffard) [inline modules/runtime/pages/performance-tuning.adoc]:
```suggestion
* In the line: `CATALINA_OPTS=`
```

**Comment by tbouffard** [comment]
**Screenshots from a3e63f4**

https://bonitasoft-bonita-doc-build_preview-pr-1852.surge.sh/bonita/2021.2/runtime/performance-tuning#jvm

image

**Pull request merged**